configs/ssg_platform_tools-vfxplatform.yaml

```diff
@@ -0,0 +1,50 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: vfxplatform
+  description: The VFX Reference Platform https://vfxplatform.com/
```

Comment:
Suggest using the following description: `The VFX Reference Platform (CY2023) https://vfxplatform.com/` since `CY2023` is met by c9s.